core-api/src/main/java/com/optimizely/ab/config/parser/JsonConfigParser.java

```diff
@@ -266,19 +266,43 @@ public ProjectConfig parseProjectConfig(@Nonnull String json) throws ConfigParse
         List<Audience> audiences = new ArrayList<Audience>(audienceJson.length());
 
         for (Object obj : audienceJson) {
-            JSONObject audienceObject = (JSONObject)obj;
+            JSONObject audienceObject = (JSONObject) obj;
             String id = audienceObject.getString("id");
             String key = audienceObject.getString("name");
             String conditionString = audienceObject.getString("conditions");
-
-            JSONArray conditionJson = new JSONArray(conditionString);
-            Condition conditions = parseConditions(conditionJson);
+            Condition conditions;
+            if (conditionString.startsWith("[")) {
```

Comment:
it would be better to use `JSONTokener` to inspect the JSON string versus directly inspecting (side-note: `charAt(0) == '['` should be preferred over `startsWith("[")`). The tokenizer can be passed to `JSONArray`/`JSONObject` constructor rather than string (it does this under the hood anyway)